### PR TITLE
fix(api-example): seeding of images

### PR DESCRIPTION
**Pull request by Itrulia**
The upload is now timing out, rest works

### CODE REVIEW

File: libs/media/api/src/lib/media-service/supported-images-validator.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const supportedMimeTypes = [
   'image/png',
   'image/gif',
   'image/jpeg',
+  'image/jpg',
```
Comment (Itrulia) [inline libs/media/api/src/lib/media-service/supported-images-validator.ts:9]: Funny enough thats the wrong mimetype for jpegs, but some images use that one

File: apps/media/src/app/app.controller.ts [hunk 2]
```diff
@@ -43,6 +43,7 @@ export class AppController {
     uploadedFile: Express.Multer.File
   ) {
     const imageId = uuidv4()
+    console.log(imageId)
```
Comment (Itrulia) [inline apps/media/src/app/app.controller.ts:46]: a console.log to see if I am hitting the controller (which I don't right now)

**Review by michael-scheurer — APPROVED** [review]
Solves the seeding on my local machine too <3

**Comment by Itrulia** [comment]
@michael-scheurer wait it works for you? This doesn't do it for me as the uploading of images takes ages.
I wanted to try a different way by spawning the server in a child process via `execa` and killing it afterwards.

**Comment by Itrulia** [comment]
@michael-scheurer ping

**Pull request merged**